### ISSUE TITLE
Define logit

### ISSUE THREAD

**Issue by rdong8** [d2l-ai/d2l-en#2645]
I don't think logits are ever formally defined anywhere in the book, they're just suddenly mentioned in 4.1 Softmax Regression.

**Comment by muzammil5539** [comment]
> I don't think logits are ever formally defined anywhere in the book, they're just suddenly mentioned in 4.1 Softmax Regression.

Logits are like the "raw scores" a neuron calculates before any activation function is applied.

Here's a simple example:

Logits = (weights × inputs) + bias

These logits are then passed through an activation function to get the final output.

# Example logits calculation
logits = (weights * inputs) + bias

# Apply activation function (e.g., sigmoid for binary classification)

output = activation(logits)
